.github/workflows/release.yaml

```diff
@@ -35,7 +35,9 @@ jobs:
           echo "GORELEASER_PREVIOUS_TAG=$(git describe --abbrev=0 --tags $(git rev-list --tags --skip=1 --max-count=1))" >> $GITHUB_ENV
           echo "GORELEASER_CURRENT_TAG=$(git describe --abbrev=0 --tags)" >> $GITHUB_ENV
       - name: "Make All"
-        run: make all
+        run: |
+          npm i -g npm@8.1.2
```

Comment:
I don't think this is necessary - an npm is built into the release of node, so it will be the same version each time. In this case:
```bash
$ curl -s https://nodejs.org/dist/index.json | jq -r '.[] | select(.version == "v16.13.2") | .npm '
8.1.2
```